### PR TITLE
nemLogging fixes for examples and specs for newer version

### PR DESCRIPTION
**Pull request by nmccready**
fixes(nemLogging): example and specs point to  angular-simple-logger browser version

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,11 +26,11 @@ See https://tombatossals.github.com/angular-leaflet-directive
 
 ## How to use it
 
-Include [angular-simple-logger](https://github.com/nmccready/angular-simple-logger) before Angular-Leaflet js files. Logger gets installed as a requirement of Angular-Leaflet with `bower install` or `npm install`.
+Include [angular-simple-logger](https://github.com/nmccready/angular-simple-logger) before Angular-Leaflet js files. Logger gets installed as a requirement of Angular-Leaflet with `bower install` or `npm install`. Note if your using the browser to load it without CommonJS (browserify, webpack) please use angular-simple-logger.js (not index.js) .
```
Comment (nmccready) [inline README.md]: Note an possible add that if you want the CommonJS version then use npm and not bower. I can not link to visionmediadebug as there is no dependency tree.

**Comment by nmccready** [comment]
@simison @tombatossals @elesdoar @jessertaylor please review. I plan on bumping to 0.9.X on this change.

**Comment by nmccready** [comment]
Anyone?

**Comment by elesdoar** [comment]
Good for me :+1:

**Comment by nmccready** [comment]
TY

**Pull request merged**